package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "navbar-mark",
+  "version": "1.0.0",
+  "description": "BundleNav for FEC",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Before this can be merged in, there will need to be some tests added here.

Reply:
Will be adding tests later, not required at this point. Script was autogenerated when initialized project. Removed script for now.